common/src/keepassPlugin/databaseInterface/KeepassIcon.cpp

```diff
@@ -37,7 +36,7 @@ QImage KeepassIcon::requestImage(const QString &uuid, QSize *size, const QSize &
     Q_UNUSED(requestedSize);
     QImage icon;
     // Check lenth and determine if the icon is a standart keepass icon (e.g. icf12) otherwise the icon is a custom one (uuid length is 32 chars)
-    if (uuid.size() != (Uuid::Length * 2)) {
+    if (uuid.size() != 32) {
```

Comment:
Would be nice to put the magic number in some define or class constant.

```suggestion
    if (uuid.size() != KeepassIcon::StandardLength) {
```

Reply:
absolutely agree